en/operations/multinode-systems.html

```diff
@@ -316,6 +317,64 @@ <h3 id="vespa-nodes-setup">Vespa nodes setup</h3>
 
 
 
+<h3 id="aws-singlenode">AWS singlenode</h3>
+<p>
+  This is a variant of the multinode install, using only one host,
+  running both a config server and the other Vespa services on the same node.
+</p>
+<ul>
+  <li>
+    Provision a node, minumim a <em>t2.large</em>. Get its hostname for use in <code>VESPA_CONFIGSERVERS</code>:
+<pre>
+$ hostname
+</pre>
+  </li>
+  <li>
+    Install Vespa:
+<pre>
+$ sudo dnf config-manager \
+  --add-repo https://raw.githubusercontent.com/vespa-engine/vespa/master/dist/vespa-engine.repo
+$ sudo dnf config-manager --enable powertools
+$ sudo dnf install -y epel-release
+$ sudo dnf install -y vespa
+$ export VESPA_HOME=/opt/vespa
+$ echo "override VESPA_CONFIGSERVERS ip-172-31-95-248.ec2.internal" | \
```

Comment:
I expect this will result in users copying it verbatim and needing to debug why this failed.
does `hostname` resolve to the same host. If so it can be replaced with `$(hostname)` and there is one less thing that may go wrong.

Reply:
I agree. otoh, this setting is often set wrong in a multi-config-server setup, so it is better that users get used to setting actual values here - so I wanted to keep it the same as the multinode setup above, but using only one name, to highlight the difference.

Reply:
sounds good